pkg/api/common/versions.go

```diff
@@ -142,13 +142,14 @@ var AllKubernetesSupportedVersions = map[string]bool{
 	"1.15.2":         false,
 	"1.15.3":         true,
 	"1.15.4":         true,
-	"1.16.0-alpha.1": true,
```

Comment:
The prerelease versions should have been set to `false` in #1972. Better late than never.